app/src/main/res/drawable/shape_status_ok.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <corners
+        android:topLeftRadius="4dp"
+        android:topRightRadius="4dp" />
+
+    <solid android:color="@color/accent" />
+
+    <size
+        android:width="100dp"
+        android:height="100dp" />
+</shape>
```

Comment:
please rename to something like `shape_status_ok_background` or `shape_status_ok_bg` so we know it is related to background.

Reply:
Fixed